taichi/llvm/llvm_offline_cache.cpp

```diff
@@ -0,0 +1,87 @@
+#include "llvm_offline_cache.h"
+
+#include "llvm/AsmParser/Parser.h"
+#include "llvm/Support/SourceMgr.h"
+#include "llvm/Support/raw_os_ostream.h"
+#include "llvm/IR/Module.h"
+
+TLANG_NAMESPACE_BEGIN
```

Comment:
According to #2198, we should at least not introduce new files with `T*_NAMESPACE_BEGIN`.